Projects/AppCore/DesignSystem/Sources/Foundation/Fonts/UILabel+.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UILabel+.swift
+//  AppCore_DesignSystem
+//
+//  Created by 마경미 on 11.06.24.
+//  Copyright © 2024 tellingUs. All rights reserved.
+//
+
+import UIKit
+
+extension UILabel {
+    public func setTextWithStyle(text: String?, fontStyle: Font.Style) {
```

Comment:
자주 사용할 것 같은 메서드인데 너무 길어서 setText는 어떨까요? 🙇🏻‍♂️

```suggestion
    public func setText(text: String?, style: Font.Style) {
```